src/contractTest/java/uk/gov/hmcts/reform/wataskmanagementapi/provider/TaskManagementGetTaskProviderTest.java

```diff
@@ -56,6 +56,10 @@
     @Autowired
     private ObjectMapper objectMapper;
 
+
```

Comment:
curious if the 2 blank lines is a styling choice?

Reply:
That's an interesting one to be completely honest there's no particular reason and the checkstyle does not enforce this rule I just find it easier to read like this, but that's just my opinion would be good to see what the other devs think it is true that it would make classes take more space on the IDE window.